src/scenes/gameuiscreen.ts

```diff
@@ -68,9 +71,23 @@ export class GameUiScreen extends Phaser.Scene {
             this.load.image(`tile${i}`, `assets/tiles/tile${i}.png`);
         }
         this.load.image("trashCan", "assets/buttons/trashcan.png");
+        // Can't get rid of error message "Object is possibly null"
+        for (let i: int = 0; i < PlayerManager.getAll().length; i++) {
+            let player: Player = PlayerManager.getAll()[i];
```

Comment:
by those 2 usages, use the const you will make above.

Reply:
Not sure what you mean, let me try